.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,46 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: 'Run tests with node ${{ matrix.node-version }}'
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - node-version: 10
+                    - node-version: 12
+                    - node-version: 14
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - uses: actions/cache@v2
+              with:
+                  path: ~/.npm
+                  key: ${{ runner.os }}-node-${{ hashFiles('package.json', 'package-lock.json') }}
+                  restore-keys: |
+                      ${{ runner.os }}-node-
+
+            - name: Install dependencies
+              run: |
+                  npm install -g grunt-cli
+                  npm install
+
+            - name: Execute test cases
+              run: grunt ci --force
```

Comment:
whats the reason for `--force`? 🙂

Reply:
I have no idea why this fails without the `--force` ... See https://github.com/luca-rath/husky/runs/1701422017

Reply:
alright - looks like this failed on travis too: https://travis-ci.org/github/massiveart/husky